bullet_train/Gemfile

```diff
@@ -13,3 +13,6 @@ gem "active_hash", github: "bullet-train-co/active_hash"
 
 # Start debugger with binding.b [https://github.com/ruby/debug]
 # gem "debug", ">= 1.0.0"
+
+# Allow users to supply content with markdown formatting. Powers our markdown() view helper.
+gem "commonmarker", ">= 1.0.0.pre"
```

Comment:
I don't think we need this here since it's in the `.gemspec`.

Reply:
Hmmmm, not sure, can you or someone from the team take over? I'm not using BT at the moment and don't have the capacity to follow-up here.

Reply:
Yep, I can take this over @vfonic. Thanks for the contribution!

Reply:
Awesome! Thank you! 🙏